src/components/expandable-row.vue

```diff
@@ -73,25 +73,15 @@ export default {
   }
 
   .button-cell {
-    font-size: 20px;
-    text-align: right;
-    line-height: normal;
+    align-self: center;
+    text-align: center;
     padding: 0px;
+    width: 30px;
 
-    a {
-      color: #888;
-      display: block;
-      width: 30px;
```

Comment:
I moved the width to the `.button-cell` so that only the button will have a background color on focus, not the whole cell.